docs/demos.md

```diff
@@ -30,7 +30,7 @@ markdown file in `docs/demos/`.
     [session](https://sched.co/Grcc)
     ([video](https://www.youtube.com/watch?v=iNlqLr9vlD4&feature=youtu.be))
   - Participants: IBM, Knative, Oracle, OpenFaaS, VMware, Microsoft, Huawei,
-    SAP, PayPal
```

Comment:
This should really remain in place I believe; PayPal was a part of that demo so it seems appropriate to leave it mentioned there

Reply:
ok - wasn't sure how much you wanted removed, so I often for all of it :-)
Fixed